src/Particle/LongRange/StructFact.h

```diff
@@ -48,6 +48,9 @@ class StructFact : public QMCTraits
   /** Constructor - copy ParticleSet and init. k-shells
    * @param lattice long range box
    * @param kc cutoff for k
+   *
+   * At least in the batched version Structure factor is _NOT_ valid
+   * after construction.
```

Comment:
It seems you are trying to describe an issue. Could you put it as an issue in github?